logstash-core/lib/logstash/runner.rb

```diff
@@ -115,6 +115,11 @@ class LogStash::Runner < Clamp::StrictCommand
     :attribute_name => "http.port",
     :default => LogStash::SETTINGS.get_default("http.port"), &:to_i
 
+  option ["--http.environment"], "HTTP_ENVIRONMENT",
```

Comment:
I think that makes sense. The only reason this is a configuration option is so that it can be set [here](https://github.com/elastic/logstash/pull/5411/files#diff-e0c0c10829ae5c5711afbe0289778cebR49) by the test suite.

It does need to be set at runner instantiation time however!